package-vst3.sh

```diff
@@ -2,18 +2,19 @@
 
 # input config
 RES_SRC_LOCATION="resources"
-PACKAGE_SRC_LOCATION="$RES_SRC_LOCATION/osx-vst2"
+PACKAGE_SRC_LOCATION="$RES_SRC_LOCATION/osx-vst3"
```

Comment:
This doesn't exist does it?
afaict it's OK to use the same resources for both VST2 and VST3 (certainly on other plugins I've built with JUCE there is no difference between `Info.plist` and `PkgInfo` between the VST2 and VST3)